helix-admin-webapp/pom.xml

```diff
@@ -79,12 +79,12 @@ under the License.
     <dependency>
       <groupId>org.codehaus.jackson</groupId>
       <artifactId>jackson-core-asl</artifactId>
-      <version>1.8.5</version>
```

Comment:
Is this version bump up necessary? 

Sometimes open source libraries aren't always backward compatible and could cause unwarranted code behavior change. So if there's no clear reason, we'd prefer to keep the version we know to be working.

Reply:
It is because "open source libraries aren't always backward compatible and could cause unwarranted code behavior change.". We have different dependency versions for Jackson-core-asl. In run time, this version difference may cause conflict and exceptions like `NoSuchClassException`. I actually would like to upgrade it to make sure only one version is used in our project to avoid conflicts.